app/decorators/prosecution_case_decorator.rb

```diff
@@ -1,18 +1,16 @@
 # frozen_string_literal: true
 
 class ProsecutionCaseDecorator < BaseDecorator
+  attr_accessor :column, :direction
```

Comment:
**not a blocker**
the name `column` in the context of this class does not clarify that it is actually a hearing column that is being referred to and the direction is the direction for hearing display.

I am not wanting to block this further but are we missing a class here - like HearingsDecorator that should know about decorating a list of hearings

Reply:
I have amended column and direction to hearings_sort_column and hearings_sort_direction.